umbra/results/c6a.metal.json

```diff
@@ -1,56 +1,54 @@
 {
     "system": "Umbra",
-    "date": "2024-02-02",
+    "date": "2024-02-05",
     "machine": "c6a.metal, 500gb gp2",
     "cluster_size": 1,
     "tags": ["C++", "column-oriented", "PostgreSQL compatible"],
-    "load_time": 432.753,
-    "data_size": 104811147008,
-    "comment": "It looks like it caches the data in memory, so the cold run results are incorrect. When I attempt to restart the server, it didn't start due to Segmentation fault.",
-    "hide": true,
+    "load_time": 224.253,
+    "data_size": 37985071698,
```

Comment:
why it is 3 times lower now?

Reply:
Minor improvements. Our original measurements already reported 47772871463 bytes. We could not reproduce your results with 104811147008 bytes.

Reply:
thanks

Reply:
confirmed 

``` sh
37522702923	db/
37522702923	total
```